maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -433,6 +433,8 @@ void cli( CliRequest cliRequest )
 
         if ( cliRequest.commandLine.hasOption( CLIManager.VERSION ) )
         {
+            // MNG-7032: Also disable colours if in batch mode
```

Comment:
This comment is redundant because we have commit messages and blames.

Reply:
Hi @michael-o – are you sure it is "unwanted"? There are numerous similar comments…

Also, I think it is helpful for readability. You do not always have all commits on your second screen. Blame might not help if there was a slight change in a later version, as blame only shows the LATEST commit of a line.

![2020-12-27T11:46:22_0001_screenshot](https://user-images.githubusercontent.com/1413391/103169106-47362a80-4839-11eb-9a62-ee0ec7f35be9.png)

Reply:
I now that there are, I think this really belongs to an SCM unless you are describing the behavior of the code. Getting "MNG-7032" git be done with 'git blame', so this information is redundant for me. Like people adding dates to their comments.

Reply:
Only if there's a huge comment required to explain what's happening based on a ticket, it might be worth adding the issue number for easy access to the discussion. In this case it is quite clear what's happing, especially if the comment is almost the same as the methodname.

Reply:
Thanks, that is helpful insight. I now see why the comment is redundant.